### DISCUSSION TITLE
[Question]: how to unban user manally?

### DISCUSSION THREAD

**Discussion by KinglyWayne** [danny-avila/LibreChat#1980] [Q&A]
### What is your question?

some user has been banned for trigger policy. How could I unban that account?

### More Details

some user has been banned for trigger policy. How could I unban that account?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
You have to find the user in the "logs" collection of your database and remove it. When the [admin panel](https://librechat.ai/) releases, you will have an interface for this to make it easier.

**Reply by Josanshuo** [reply]
I have encounter the same issue today, however, the logs collection is empty. I also notice there is a banner collection, which is also empty in my database.

For some context, I updated my server from 0.7.3 to 0.7.7 using repo version. One of the account is working initially, but today it suddenly show the error. 

Is there another way to remove the ban for certain user?

**Comment by rollingcookies** [comment]
So, today my account in a local installation (without Docker) LibreChat locked up, I can't access it now. I deleted all the entries in the 'logs' collection (MongoDB>Librechat base). The server has been rebooted. I also removed all restrictions in the .ENV settings:
```
#========================#
# Moderation             #
#========================#

OPENAI_MODERATION=false
OPENAI_MODERATION_API_KEY=
# OPENAI_MODERATION_REVERSE_PROXY=

BAN_VIOLATIONS=false
BAN_DURATION=1
BAN_INTERVAL=9999999
# BAN_DURATION=1000 * 60 * 60 * 2
# BAN_INTERVAL=20

LOGIN_VIOLATION_SCORE=0
REGISTRATION_VIOLATION_SCORE=0
CONCURRENT_VIOLATION_SCORE=0
MESSAGE_VIOLATION_SCORE=0
NON_BROWSER_VIOLATION_SCORE=0
TTS_VIOLATION_SCORE=0
STT_VIOLATION_SCORE=0
FORK_VIOLATION_SCORE=0
IMPORT_VIOLATION_SCORE=0
FILE_UPLOAD_VIOLATION_SCORE=0
# ILLEGAL_MODEL_REQ_SCORE=0

LOGIN_MAX=9999999
LOGIN_WINDOW=9999999
REGISTER_MAX=9999999
REGISTER_WINDOW=9999999

LIMIT_CONCURRENT_MESSAGES=false
CONCURRENT_MESSAGE_MAX=9999999

LIMIT_MESSAGE_IP=false
MESSAGE_IP_MAX=9999999
MESSAGE_IP_WINDOW=9999999

LIMIT_MESSAGE_USER=false
MESSAGE_USER_MAX=9999999
MESSAGE_USER_WINDOW=9999999

LIMIT_LOGIN_IP=false

LIMIT_IMPORT_IP=false
IMPORT_IP_MAX=9999999
LIMIT_IMPORT_USER=false
IMPORT_USER_MAX=9999999
LIMIT_FORK_IP=false
FORK_IP_MAX=9999999
LIMIT_FORK_USER=false
FORK_USER_MAX=9999999


#========================#
# Registration and Login #
#========================#

ALLOW_EMAIL_LOGIN=true
ALLOW_REGISTRATION=true
ALLOW_SOCIAL_LOGIN=false
ALLOW_SOCIAL_REGISTRATION=false
ALLOW_PASSWORD_RESET=false
# ALLOW_ACCOUNT_DELETION=true # note: enabled by default if omitted/commented out
ALLOW_UNVERIFIED_EMAIL_LOGIN=true

SESSION_EXPIRY=((1000 * 60 * 60 * 24) * 7) * 10000000000000000
REFRESH_TOKEN_EXPIRY=((1000 * 60 * 60 * 24) * 7) * 10000000000000000

===========================
```

Cleared cookies in the browser. Tried running in DEV mode:

set NODE_PATH=C:\Program Files\nvm4m\nodejs
"%NODE_PATH%\node.exe" "%NODE_PATH%\node_modules\npm\bin\npm-cli.js" run backend:dev

But nothing helps to solve the problem! Help, please! There are valuable data left in the chats!

**Comment by 3750gustavo** [comment]
I managed to get unbanned running a few commands on docker desktop:

### 1. Open the MongoDB Shell
Open the terminal for your `chat-mongodb` container (via Docker Desktop Exec or `docker exec -it chat-mongodb mongosh`) and run:
```bash
mongosh
```

### 2. Locate and Switch to the Database
Run this to see your database names:
```js
show dbs
```
Look for `LibreChat` or `librechat`. Switch to it (replace with your actual DB name):
```js
use LibreChat
```

### 3. Run the Unban Command
Replace `YOUR_USER_ID` with the ID found in your logs (e.g., `66391f46j665d5d8b013f5dc`). This command removes all ban-related flags from your user document:

```js
db.users.updateOne(
  { _id: ObjectId("YOUR_USER_ID") },
  { $unset: { bannedAt: "", bannedUntil: "", banReason: "", banned: "", banIp: "" } }
)
```
*Note: If your ID is stored as a string, remove the `ObjectId()` wrapper and just use quotes around the ID.*

### 4. Optional: Prevent future auto-bans
If you are being banned due to provider errors, edit your `.env` file to disable the automated system:
```bash
BAN_VIOLATIONS=false
ILLEGAL_MODEL_REQ_SCORE=0
```

and also make the other scores 0.

Restart your containers after saving the `.env` changes.
